.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,36 @@
+name: Pull Request Validation
+
+on:
+  pull_request:
+    branches:
+      - trunk
+    types:
+      - labeled
+      - unlabeled
+      - opened
+      - reopened
+      - synchronize
+      - edited
+
+env:
+  LABELS: ${{ join( github.event.pull_request.labels.*.name, ' ' ) }}
+
+jobs:
+  check-type-label:
+    name: Check [Type] Label
+    runs-on: ubuntu-latest
+    steps:
+      - if: contains( env.LABELS, '[Type]' ) == false
+        run: exit 1
```

Comment:
@eugene-manuilov do you think we can add a message with the exit?

Reply:
@JustinyAhin I think echoing messages is redundant for those checks since it's clear what is wrong by looking at the title of the job.

Reply:
Make sense yes.